rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -324,8 +323,8 @@ async fn smoke_test(ctlr_tx: CtlrTx) -> Result<()> {
         .await
         .context("Failed to get zip file metadata")?
         .len();
-    if zip_len == 0 {
-        bail!("Exported log zip has 0 bytes");
+    if zip_len <= 22 {
+        bail!("Exported log zip just has the file header");
```

Comment:
This would catch the bug where we leave an empty zip and the test gives a false positive